### PR TITLE
Refactor/vendor proposal modifications

### PR DESCRIPTION
**Pull request by IanFonzie**
This PR closes issue: [DM-1699]

Includes tests? [N]
Updated docs? [N]

Proposed changes:
- Mostly wording mentioned in DM-1699
- Also adds a new tab to the vendor opportunity view that loads the TWU terms and conditions content

### CODE REVIEW

File: src/front-end/typescript/lib/pages/opportunity/team-with-us/view.tsx [hunk 1]
```diff
@@ -75,6 +82,7 @@ export type InnerMsg =
         api.ResponseValidation<TWUOpportunity, string[]>,
         TWUProposalSlim | null,
         api.ResponseValidation<Content, string[]>,
+        api.ResponseValidation<Content, string[]>,
```
Comment (SteveChapmanBCDX) [inline src/front-end/typescript/lib/pages/opportunity/team-with-us/view.tsx:85]: Is this change required here?  It looks like a duplication of line 84.

**Review by SteveChapmanBCDX — APPROVED** [review]
Everything looks good.  Thanks for getting this done right away.

File: src/front-end/typescript/lib/pages/opportunity/team-with-us/view.tsx (hunk 1 above)
Comment (IanFonzie) [reply src/front-end/typescript/lib/pages/opportunity/team-with-us/view.tsx:85]: yup, it's necessary. We're specifying what our messages should look like and those two happen to share a similar type (because they're both content).
